orbit/cmd/desktop/desktop.go

```diff
@@ -3,19 +3,50 @@ package main
 import (
 	"crypto/tls"
 	_ "embed"
+	"encoding/json"
 	"fmt"
+	"github.com/fleetdm/fleet/v4/pkg/open"
+	"github.com/getlantern/systray"
+	"io/ioutil"
 	"log"
 	"net/http"
 	"net/url"
 	"os"
+	"strings"
 	"time"
-
-	"github.com/fleetdm/fleet/v4/pkg/open"
-	"github.com/getlantern/systray"
 )
 
 var version = "unknown"
 
+type Response struct {
```

Comment:
NOTE: This won't be necessary once we export some "Client" methods in `server/service`. (Similar to what we do for `fleetctl` code that needs to access Fleet API from Go code.)

I'll be pushing a separate PR for that.

Reply:
Sounds goo, thanks @lucasmrod!